cardano-node/src/Cardano/Node/Handlers/TopLevel.hs

```diff
@@ -64,6 +65,7 @@ import           System.IO
 --
 toplevelExceptionHandler :: IO a -> IO a
 toplevelExceptionHandler prog = do
+    outPutStrLn "Starting"
```

Comment:
Only this gets logged when the test fails.  This tells me no exceptions were intercepted.